### ISSUE TITLE
Admonitions or Call-outs?

### ISSUE THREAD

**Issue by DutchPete** [valentine195/obsidian-admonition#224]
Will Admonitions eventually cease to exist and Call-outs will be the only way? If not, what would be a reason for a person to use Admonitions instead of Call-outs or vice versa? I have not been able to figure this out.

**Comment by I-Pch** [comment]
The native callouts are not really customisable at least, at the moment (without potentially having to dig into the CSS)... but this might evolve as time passes.
Admonitions on the other hand, still allows you to easily create customised callouts and use them as you please 😊 (with or without a title, with a specific icon, color, etc...) .

So, as far as I'm concerned, the plugin is still very very useful as it enhances the native callouts, making them more flexible without too much troubles 😊 (which is a big plus in terms of UX and UI ).

**Comment by DutchPete** [comment]
@I-Pch Hi there, thanks so much for your feedback.

I use Admonitions a lot, but struggle a bit with the interaction between them and callouts.
When you say “Admonitions on the other hand, still allows you to easily create customised callouts and use them as you please”, what do you mean?

And Admonition starts with e.g. ` ```ad-note`, whereas a callout starts, like a blockquote, with `>`. To me, they are 2 entirely different things.

To put it a different way, is there any reason to not use Admonitions anymore and to switch to callouts? Makes sens if the former are on the way out. OTOH, if Admonitions are not on the way out, what is the point of using callouts?

**Comment by I-Pch** [comment]
Hi @DutchPete 😊 !

> When you say “Admonitions on the other hand, still allows you to easily create customised callouts and use them as you please”, what do you mean?

I was speaking about the the functionalities of the plugin 😅, not the Admonitions in themselves (and lol, don't worry, I'm also a bit confused here 😇 . I'm not sure how to call what anymore 😅 ) 

> And Admonition starts with e.g. ```ad-note, whereas a callout starts, like a blockquote, with >. To me, they are 2 entirely different things.

Not entirely 😊 ... I'm not a dev' but if I understood correctly what happened, the core functionalities of the original plugin have been integrated into Obsidian (which seems to have solved some limitations/bugs the plugin had) but only with basic callouts (note, quotes, faq, etc...), making the whole feature now more powerful, stable and perfectly usable alone.

But the plugin is still there for users wanting more than the basic callouts 😊 ... 

E.g.: If you create a "custom" callout (in other words, an admonition (I guess)), you can use it in you notes with a ` ```ad-something `  but as the plugin and Obsidian work now together, you can also use the Admonition in your notes using the "callouts " syntax (`> [!something]`) 

I think that what I'm trying to say (I'm sorry, I didn't get much sleep last night) is that no, I don't think there's any reason for you to specifically switch to callouts if you don't want to ... But, they are definitively there if you wish 😊.

**Comment by valentine195** [comment]
Code block Admonitions aren’t going away, but, in general, I would recommend using callouts going forward as they have several advantages:

1. They are valid markdown, so they are more future proof as they will fall back to a block quote. 
2. Links, tags and footnotes work in them and are added to Obsidian’s cache
3. They are native, so they do not deal with some of the bugs or workarounds I had to implement (embeds and checkboxes, notably)
4. They just work on publish
5. I am not going to spend coding effort trying to compete with them - code blocks are here to stay but they are staying “as-is”

**Comment by I-Pch** [comment]
> I would recommend using callouts going forward

Just to be clear though 😇 : When you talk about callouts here, you mean the "admonitions/custom callouts" we can create through the plugin but used with the native Obsidian callouts syntax (`> [!something]`), it that it 😇 ?

(Sorry, I know this is probably stupid but this is what's actually confusing me 😅 )

**Comment by valentine195** [comment]
> Obsidian callouts syntax

correct, I would recommend using the new Obsidian callout syntax. The only real advantage the code block admonitions have over them is the ability to render without a title and the ability to edit without the `>

**Comment by I-Pch** [comment]
Thank you very much @valentine195 😁 !
It's all clear now 🙌 !

**Comment by DutchPete** [comment]
OK, it is clear for me now.

Regarding the Admonition ➙ Callout conversion via the button your plug-in provides, I have a number of Admonitions in which the title consists of `[some title](URL`): will that screw up conversion?

I also have Admonition with the title `some title⁺[S](URL)`: any conversion problems foreseen?
